apps/studio/components/interfaces/DataWarehouse/CreateWarehouseCollection.tsx

```diff
@@ -103,6 +127,21 @@ export const CreateWarehouseCollectionModal = () => {
                   </FormItemLayout>
                 )}
               />
+              <FormField_Shadcn_
```

Comment:
might need some gap-y-1 maybe between the 2 fields

![image](https://github.com/user-attachments/assets/283ad810-8e93-41d3-86c2-509db323d745)

Reply:
🫡
![CleanShot 2024-09-25 at 18 50 07@2x](https://github.com/user-attachments/assets/7a939c45-8de8-4302-a62a-e898816ae24a)